src/99-appendices/10-file-collections.md

```diff
@@ -0,0 +1,50 @@
+# Appendix X: File collections
+
+To give context to the use of [entity-linked file collections](./02-common-principles.md#file-name-structure), applications concerned with this
+common principle are listed by descriptive tables under their respective modalities.
+
+Note that the tables in this appendix catolog applications where the use of 
+a file collection is REQUIRED.
+
+Please keep this list of linking entities up-to-date with the file collections included
+in this appendix:
```

Comment:
I think a bit of information about what constitutes a linking vs. (let's say) separating entity should go here. Basically the idea that some entities imply a file collection while others do not (e.g., `run`).

Reply:
Thank you Taylor, addressed in https://github.com/bids-standard/bids-specification/pull/668/commits/e80ede6842262ecc8f051ecaa30e625eb3288967.